os-appium-tests/tests/helpers/NativeAlert.ts

```diff
@@ -0,0 +1,62 @@
+import {DEFAULT_TIMEOUT, DEFAULT_TIMEOUT_INTERVAL} from '../constants';
+
+const SELECTORS = {
+    ANDROID: {
+        ALERT_BUTTON: '*//android.widget.Button[@text="{BUTTON_TEXT}"]',
+        ALERT_DIALOG: '*//android.widget.LinearLayout[@resource-id="com.android.packageinstaller:id/dialog_container"]', // Sandra
+        ALERT_TITLE: '*//android.widget.TextView[@resource-id="com.android.packageinstaller:id/permission_message"]', // ANDROID 9
+        // ALERT_TITLE: '*//android.widget.TextView[@resource-id="android:id/alertTitle"]', //BEFORE ANDROID 9
```

Comment:
So we should use selector `ALERT_TITLE: '*//android.widget.TextView[@resource-id="android:id/alertTitle"` is we are testing versions before Android 9? If so, can we know the Android version using to tests on runtime and change the selector content dynamically? 

This would allow us to quickly change the .ipa without having to refactor the code :)

Reply:
Great suggestion, that refactor should be done at the framework file @ines-sousa